Makefile

```diff
@@ -140,11 +140,11 @@ ifneq ($(GITHUB_BASE_REF), false)
 	git diff --name-only FETCH_HEAD | grep '^requirements/' || exit 0 && $(MAKE) deps
 endif
 
+resetdb:
+	docker-compose rm --stop --force db
+	docker-compose up -d db
```

Comment:
If we add a named volume, we'll need to delete it here.

Reply:
So the named volume should hold the DB after it was imported and initialized. To me that extra redirection is not necessary, because `docker-compose down` doesn't kill the container contents. The only way DB content can be killed if somebody explicitly removes the containers explicitly with `rm` command. Why would we want to preserve the DB content in this case?

Reply:
> The only way DB content can be killed if somebody explicitly removes the container with rm command

Or if the user uses `docker-compose down`.

As you want, really. I offered this as a suggestion. I know I do it in my projects usually and it works for me, but it doesn't have to be useful for everyone.

Reply:
To me `docker compose down` means 'free all useless disk space taken by containers of this project' (and `docker compose down -v` means free all disk space taken by containers of this project'). I like being able to remove quickly everything except the thing that would be long to recreate.

Reply:
Can you confirm that `docker-compose down` really removes the containers? Because in my setup it doesn't happen. I will check out once more right now, but I am pretty sure the behavior is that compose just issues `stop` and not `rm`.

Or do you mean that `docker-compose down` cleans up volumes, and this is what you want? But then you will need to reset the DB and run migrations anyway.

Reply:
I was wrong. The compose setup survives reboot, but all containers are removed when `down` is invoked. So we need a named volume.

Reply:
yep. `docker-compose down = docker-compose stop + docker-compose rm`, roughly

Reply:
Added named volume removal with explicit `docker` command, because `docker-compose` doesn't support this https://stackoverflow.com/questions/45511956/remove-a-named-volume-with-docker-compose